worlds/subnautica/options.py

```diff
@@ -112,8 +112,7 @@ def get_pool(self) -> typing.List[str]:
 
 
 class SubnauticaDeathLink(DeathLink):
-    """When you die, everyone dies. Of course the reverse is true too.
-    Note: can be toggled via in-game console command "deathlink"."""
+    __doc__ = DeathLink.__doc__ + "\n    Note: can be toggled via in-game console command \"deathlink\"."
```

Comment:
Presumably you meant this instead?
```suggestion
    __doc__ = DeathLink.__doc__ + "\n\n    Note: can be toggled via in-game console command \"deathlink\"."
```

Reply:
original didn't have a line break either

Reply:
True, but I kind of like the line break.